### PR TITLE
Feature/mobile

### PR DESCRIPTION
**Pull request by Ddiiro**
- Created a contact page.
- Validated the fields on the contact page.
- Arranged the layout using flexbox.
- Created a form with a valid endpoint.
- Updated the CSS files in the project.
- Cleaned the project of any linter errors.

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Ddiiro  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that has no commit history from your pair programming partner. Kindly reach out to your partner so that they can contribute to your project. I would like to review your project but because of this, I can't proceed further.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -210,27 +213,33 @@ <h3>Languages</h3>
...
-   <textarea placeholder="Write your message here" name="" id="" cols="30" rows="10"></textarea>
-
-<button  class="contact-button"><a href="">Get in touch</a></button>
-
-<div class=""><img src="Images/Shape.png" alt=""></div>
+<form action="https://formspree.io/f/xgebkzlz" method="post">
+   <input name="Name" placeholder="Yeremias" type="text" maxlength="30" required>
+   <input name="Email" placeholder="natajayanj14@gmail.com" type="email" required>
+   <textarea placeholder="Write your message here" name="Message" id="" cols="30" rows="10" maxlength="500" type="text" required></textarea>
+<button type="submit" id="contact-button"><a href="">Get in touch</a></button>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] I have noticed that your form is not able to submit 😞 . I have noticed that you have used the `a` tag inside the `button` tag. This is not Semantically correct and therefore the form won't be submitted. Kindly consider removing it and verify the form submission ☺️

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @Ddiiro 🙋‍♂️
Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  
However, there are some issues mostly on the button for [semantic HTML](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjwyP2CrLD8AhUZi_0HHdAIB8UQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU) practice and font colors. 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes ♻️ 
- The experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. If you do not incorporate control in the speed of the scroll, it can quickly turn problematic with content whizzing past or taking forever to move 😞 . Kindly ensure that the horizontal scroll is removed.
- In addition, i have realized the works section cards are not having the required space between them. We need to stick to the Figma design as much as possible and for this cas, i suggest you apply the space between the cards 👍 .

Video demo 😎 
https://clipchamp.com/watch/VPXhdKyO5UC

https://user-images.githubusercontent.com/99394374/220582548-a8e60e96-b0e8-481a-868a-439b9cfebf9f.mp4



## Optional suggestions
- [ ] [OPTIONAL]  Consider the use of [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements)
If you notice for form [accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility) reasons wrapping the inputs in the `<fieldset>` is more semantic. Kindly make the form more accessible to screen readers such us [Jaws](https://www.freedomscientific.com/products/software/jaws/) and [NVDA](https://www.nvaccess.org) by using `<fieldset>`.



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: css/contact.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+#contact h2 {
+  position: relative;
+  width: 100%;
+  height: 56px;
+  left: 5%;
+  top: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
```
Comment (shella12) [inline css/contact.css]: - [ ] Great job fixing the submit button, now we can see the form submission in action, and a great job working on your horizontal scroll, I can clearly see the difference. But there is still a little bit of horizontal scrolling happening. Please make sure to fix it and make sure your sections take up the fullscreen to make it look like the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=wgGEhimPzdFHgsEn-0) as it is the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) to stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=wgGEhimPzdFHgsEn-0)

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

https://user-images.githubusercontent.com/44798044/220824780-225115bf-e08d-4527-9e9c-124e1275d635.mov

File: css/contact.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  color: #b3bac5;
+  padding-left: 10px;
+}
+
+.bottom {
+  order: 4;
+  position: relative;
+  top: 20vh;
+  left: 18vh;
+}
```
Comment (shella12) [inline css/contact.css:140]: - [ ] [OPTIONAL] You have done a great job adding button interactions, they are really important to enhance user experience however please make sure to implement button interaction according to the Figma design. Currently, your hover and presses state interaction have the same background (hover state should have a `background-color:#6070FF;`). Optionally you can always add `box shadow` to the `button` if you feel like hover interaction makes your button blend into the background 

**_Figma design_** 

Screenshot 2023-02-23 at 10 08 56 AM

_**Your design**_


https://user-images.githubusercontent.com/44798044/220826109-eafc2433-37cf-40e3-8c26-5c8cc781f439.mov

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @Ddiiro , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- Fixed submit button ✅   

 ## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Ddiiro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — COMMENTED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @Ddiiro,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

All linter checks passed ✔️
Descriptive pull request ✔️
Used GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**